plugin/trino-postgresql/src/test/java/io/trino/plugin/postgresql/TestPostgreSqlConnectorTest.java

```diff
@@ -732,14 +733,10 @@ public void testTimestampColumnAndTimestampWithTimeZoneConstant()
     @Override
     public void testCaseSensitiveDataMapping(DataMappingTestSetup dataMappingTestSetup)
     {
-        try {
-            super.testCaseSensitiveDataMapping(dataMappingTestSetup);
-        }
-        catch (AssertionError ignored) {
-            // TODO https://github.com/trinodb/trino/issues/3645 - PostgreSQL has different collation than Trino
-            assertThatThrownBy(() -> super.testCaseSensitiveDataMapping(dataMappingTestSetup))
-                    .hasStackTraceContaining("not equal\nActual rows");
-        }
+        // TODO https://github.com/trinodb/trino/issues/3645 - PostgreSQL has different collation than Trino
+        assertThatThrownBy(() -> super.testCaseSensitiveDataMapping(dataMappingTestSetup))
```

Comment:
If it not fail then you will get assertion error

Reply:
So we don't know whether the test fails or does not fail?
As it was, it was pointless -- we could simply writen an unconditional `throw new SkippedException`, if we cannot be more strict.

Reply:
> unconditional throw new SkippedException

sounds good.